### ISSUE TITLE
Autoformat doesn't apply to Ktlint Code Style

### ISSUE THREAD

**Issue by spipau** [JLLeitschuh/ktlint-gradle#298]
After upgrading to Version 9.0.0 and changing my Gradle implementation Ktlint fails the build after:

1. Upgraded to V9.0.0.
1. Change Gradle implementation as described in readme.
1. Executed `ktlintApplyToIdea` Gradle task and restarted IDE.
1. Executed `ktlintFormat` and fixed the remaining WildCard Errors with autoformat after `ktlintCheck` run.
1. Run again `ktlintCheck` and the following error can't be resolved with auto-format:

`... HelperExtensions.kt:3:1: Imports must be ordered in lexicographic order without any empty lines in-between`

The imports in `HelperExtensions.kt`:

```
import android.content.Context
import android.graphics.Bitmap
import android.graphics.Color.BLACK
import android.graphics.Color.WHITE
import android.os.Bundle
import android.os.Handler
import android.util.Base64
import android.view.View
import android.view.inputmethod.InputMethodManager
import android.widget.EditText
import androidx.annotation.IdRes
import androidx.fragment.app.FragmentActivity
import androidx.lifecycle.MutableLiveData
import androidx.navigation.NavController
import androidx.navigation.NavDirections
import androidx.navigation.NavOptions
import androidx.navigation.fragment.FragmentNavigator
import at.risedev.coffeecash.core.R
import at.risedev.coffeecash.core.custom.ErrorDialogFragment
import com.google.zxing.BarcodeFormat
import com.google.zxing.MultiFormatWriter
import com.google.zxing.WriterException
import com.google.zxing.common.BitMatrix
import timber.log.Timber
import java.math.BigDecimal
import java.util.Locale
```

**Comment by Tapchicoma** [comment]
You are affected by this ktlint issue: https://github.com/pinterest/ktlint/issues/527

**Comment by spipau** [comment]
I now fixed this issue in version 9.1.1 by adding the following to my `.editorconfig` (root folder):

```
[*.{kt,kts}]
disabled_rules=import-ordering
```
